front-end/src/screens/Settings/email.tsx

```diff
@@ -1,21 +1,25 @@
 import React, { useState, useContext, useEffect } from 'react'
 
+import { Icon } from 'semantic-ui-react'
 import Button from '../../ui-components/Button'
 import { Form } from '../../ui-components/Form'
 import { useChangeEmailMutation } from '../../generated/graphql'
-import { UserDetailsContext } from '../../context/UserDetailsContext'
-import { NotificationContext } from '../../context/NotificationContext';
-import { NotificationStatus } from '../../types';
+import { NotificationContext } from '../../context/NotificationContext'
+import { NotificationStatus } from '../../types'
 
-const Email = (): JSX.Element => {
+interface Props {
+	profileEmail?: string | null | undefined
```

Comment:
Should we have rather part of the `UserDetailsContext` as it was before so that we avoid props drilling?